File: Secao03 - Desvendando Algoritmos/Exercicios em Python/Exercicio02.py
# -*- coding: utf-8 -*-
print("Calculo de estoque médio de peças.\n")
qnt_minima = int(input("Informe a quantidade minima de peças: "))
qnt_max = int(input("Informe a quantidade maxima de peças: "))

soma = qnt_minima + qnt_max
estoque_medio = soma / 2

print("Temos um estoque medio de {0} peças.".format(estoque_medio))

File: Secao03 - Desvendando Algoritmos/Exercicios em Java/Exercicio02.java
package br.com.udemy.secao03;

import java.util.Scanner;

public class Exercicio02 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		int qnt_maxima, qnt_minima, soma;
		float estoque_medio;
		
		System.out.print("Informe a quantidade minima de peças: ");
		qnt_minima = teclado.nextInt();
		System.out.print("Informe a quantidade maxima de peças: ");
		qnt_maxima = teclado.nextInt();
		System.out.println(" ");
		
		soma = qnt_minima + qnt_maxima;
		estoque_medio = soma / 2;
		
		System.out.print("O estoque médio é de " + estoque_medio + " peças.");
		
		teclado.close();

	}

}
